src/stylesheets/components/Auth/_auth-alert.scss

```diff
@@ -0,0 +1,26 @@
+.auth-alert {
+  backdrop-filter: blur(rem(54.3656px));
```

Comment:
@rafayet-monon I think for long precision digit like this `54.3656px` we can round up or cut it out 😛